english/00_PortBlair's Emerald blue/PortBlairVoyage.md

```diff
@@ -0,0 +1,15 @@
+# I landed on Port Blair on **26th Jan** 2008
+
+#### Why this day is special ??
+######  [Republic day of India](https://en.wikipedia.org/wiki/Republic_Day_(India))
+
+##### I was on the ship named MV Nicobar. See [here](https://en.wikipedia.org/wiki/Republic_Day_(India)) for more details about ship.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.